resources/public/api/conf/2.0/application.yaml

```diff
@@ -862,7 +862,7 @@ paths:
       description: >
         Get all arrivals that have been updated within the past 31 days.
         
-        You can filter for arrival notifications updated after and/or before a specific date and time and/or submitted with a specified EORI.
+        You can filter for arrival notifications updated after and/or before a specific date and time along with/or not, a specified EORI.
 

```

Comment:
Same issues as above.

Suggestion:
You can filter for arrival notifications updated before and/or after a specific date and time, and optionally filter for arrival notifications with a specific EORI.

Reply:
Yeah, sorry thought I'd changed that!